packages/fast-components-react-msft/src/carousel/carousel.props.ts

```diff
@@ -70,6 +70,16 @@ export interface CarouselHandledProps extends CarouselManagedClasses {
      * The carousel items
      */
     items: CarouselSlide[];
+
+    /**
+     * The carousel next flipper
+     */
```

Comment:
These receive the event object, don't they?

Reply:
An initial request was to continue managing the rotation, but I think if someone provides their own flippers they can technically fully control that functionality and advance the slide as they so desire. I can pull the callback support from here (assuming that's what you're inferring :))

Reply:
I think he's saying the the type signature should be `onClick: (e: React.MouseEvent<HTMLElement>) => void, className?: string) => React.ReactNode`